server/src/main/java/com/thoughtworks/go/server/service/BackupService.java

```diff
@@ -85,9 +85,11 @@
     private final Database databaseStrategy;
     private volatile ServerBackup runningBackup;
     private static final String CONFIG_BACKUP_ZIP = "config-dir.zip";
+    private static final String WRAPPER_CONFIG_BACKUP_ZIP = "wrapper-config-dir.zip";
 
     private static final String CONFIG_REPOSITORY_BACKUP_ZIP = "config-repo.zip";
     private static final String VERSION_BACKUP_FILE = "version.txt";
+    public static final String WRAPPER_CONFIG_DIR = "wrapper-config";
```

Comment:
For the correct location of the wrapper config dir see: https://wrapper.tanukisoftware.com/doc/english/props-envvars.html#default-definitions